src/data/_contributors.yaml

```diff
@@ -195,6 +195,15 @@ ericbidelman:
   description:
     en: 'Engineer @ Google working on web tooling: Headless Chrome, Puppeteer, Lighthouse'
 
+jbingham:
+  name:
+    given: Jonathan
+    family: Bingham
+  github: jonathanbingham
```

Comment:
My github and twitter user names are backwards. github is jbingham, twitter is jonathanbingham.

Reply:
Ooopsy. 😳